mailtrap.gemspec

```diff
@@ -19,6 +19,9 @@ Gem::Specification.new do |spec|
   spec.metadata['changelog_uri'] = 'https://github.com/railsware/mailtrap-ruby/blob/main/CHANGELOG.md'
   spec.metadata['rubygems_mfa_required'] = 'true'
 
+  spec.add_dependency 'mail'
```

Comment:
I don't think we need to add these as explicit dependencies if you can still use the gem without these.

Besides, anyone using actionmailer would have them as actionmailer's dependencies, already.

Reply:
that's true. So are we all good to move both to `Gemfile` ?
@leonid-shevtsov @IgorDobryn

Reply:
moved to Gemfile then